doc/monitoring/metrics_reference.rst

```diff
@@ -773,6 +773,9 @@ Shows the current state of a replica set node in regards to leader election.
         *   -   ``tnt_election_term``
             -   Current election term.
 
+        *   -   ``tnt_election_leader_idle``
+            -   Current idle for elected leader.
```

Comment:
Let's add more useful description: "time in seconds since the last interaction with the known leader"